src/ideation/ideation.controller.ts

```diff
@@ -1,37 +1,62 @@
-import { Controller, Get, Param } from "@nestjs/common";
+import {
+    Controller,
+    Get,
+    Param,
+    Post,
+    Body,
+    Patch,
+    Delete,
+} from "@nestjs/common";
 import { IdeationService } from "./ideation.service";
-// import { CreateIdeationDto } from "./dto/create-ideation.dto";
-// import { UpdateIdeationDto } from "./dto/update-ideation.dto";
+import { CreateIdeationDto } from "./dto/create-ideation.dto";
+import { UpdateIdeationDto } from "./dto/update-ideation.dto";
 
 @Controller("voyage")
 export class IdeationController {
     constructor(private readonly ideationService: IdeationService) {}
 
-    // @Post()
-    // create(@Body() createIdeationDto: CreateIdeationDto) {
-    //     return this.ideationService.create(createIdeationDto);
-    // }
-
-    // @Get()
-    // findAll() {
-    //     return this.ideationService.findAll();
-    // }
+    @Post(":id/project/:tId")
+    create(
+        @Param("id") id: string,
+        @Param("tId") tId: string,
+        @Body() createIdeationDto: CreateIdeationDto,
+    ) {
+        return this.ideationService.create(createIdeationDto);
+    }
 
+    @Get(":id/project")
```

Comment:
Unless I'm misunderstanding, this shouldn't be here. The getProjectIdeas is going to fetch everything.

Reply:
no, don't think we need an endpoint for fetching project ideas for ALL teams, which is what this is doing

I also suggest using more descriptive names like teamId instead of tId

also same for `async getProjectIdeas(id: number) {`

we have too many different ids